.env

```diff
@@ -1,2 +1,5 @@
 DATABASE_URL=postgres://postgres:postgres@localhost:5432/game-night-db
 SQLX_OFFLINE=true
+
+# email
+EMAIL_SENDER=noreply@gamenight.gay
```

Comment:
We probably need an SPF record. Need to check SendGrid and see if they suggest one.

Reply:
Nope. Looks like SendGrid has this handled.
https://docs.sendgrid.com/ui/account-and-settings/spf-records